src/index.ts

```diff
@@ -0,0 +1,78 @@
+const inputElement = document.getElementById("commentInput");
+const formElement = document.getElementById("inputForm");
+const ulElement = document.getElementById("commentsList");
+
+//submit form
+if (formElement !== null) {
+    formElement?.addEventListener("submit", (event) => {
+        event.preventDefault();
+        if (inputElement !== null) {
+            const newMsg = (inputElement as HTMLInputElement).value;
```

Comment:
you cast inputElement many times in this code.
cast it once

Reply:
cast = as HTMLInputElement ?
if so,
It need to be typed